README.md

```diff
@@ -53,6 +53,12 @@ You will also find detailed [guides](./docs/Guides.md) and [frequently asked que
 
 ### Prerequisite: Bitcoin Core
 
+We chose to rely as much as possible on bitcoin core, to interface with and monitor the blockchain and also to manage onchain funds (eclair does not include an onchain wallet, channel opening transactions are funded by your bitcoin core node, and channel closing transactions return funds to your bitcoin core node).
+
+This means that we have strong requirements on how your bitcoin node is configured (see below), and that you must backup your bitcoin core wallet as well as your eclair node (see https://github.com/ACINQ/eclair/edit/master/README.md#configure-bitcoin-core-wallet)
+
+But is also means that instead of re-implemeting them, we benefit from all the verifications and optimisations (including fee management with RBF/CPFP, ...) that are provided by bitcoin core, which is much better from a security point of view. We did have to implement some of the bitcoin protocol, but in the context of verifying data provided by bitcoin core.
```

Comment:
nit:

```suggestion
But is also means that instead of re-implementing them, we benefit from all the verifications and optimisations (including fee management with RBF/CPFP, ...) that are provided by bitcoin core, which is much better from a security point of view. We did have to implement some of the bitcoin protocol, but in the context of verifying data provided by bitcoin core.
```